lib/routes/bestblogs/feeds.ts

```diff
@@ -0,0 +1,121 @@
+import { Route } from '@/types';
+import ofetch from '@/utils/ofetch';
+import { parseDate } from '@/utils/parse-date';
+
+export const route: Route = {
+    path: '/feeds/:category?',
+    categories: ['programming'],
+    example: '/bestblogs/feeds/featured',
+    parameters: { category: 'the category of articles. Can be `programming`, `ai`, `product`, `business` or `featured`. Default is `featured`' },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['bestblogs.dev/feeds?featured=y'],
+            target: '/feeds/featured',
+        },
+        {
+            source: ['bestblogs.dev', 'bestblogs.dev/feeds', 'bestblogs.dev/feeds?featured=y'],
+            target: '/feeds/featured',
+        },
+        {
+            source: ['bestblogs.dev/feeds?category=:category'],
+            target: '/feeds/:category',
+        },
+    ],
+    name: 'BestBlogs.dev文章列表',
```

Comment:
```suggestion
    name: '文章列表',
```
No need to include namespace's `name` again